CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 
   **BUG FIXES**
   - [#212](https://github.com/Datatamer/unify-client-python/issues/212) Sped up slow running tests
+  - [#235](https://github.com/Datatamer/unify-client-python/issues/235) Making `AttributeCollection` retrieve attributes directly instead of by streaming
```

Comment:
@juliamcclellan can you rebase on `master` so that this goes into `0.9.0-dev`? we already bumped the version on `master`